easybuild/easyconfigs/c/CPLEX/CPLEX-22.10-GCCcore-11.2.0-Python-3.9.6.eb

```
@@ -0,0 +1,26 @@
+name = 'CPLEX'
+version = '22.10'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.ibm.com/analytics/cplex-optimizer'
+description = """IBM ILOG CPLEX Optimizer's mathematical programming technology enables
+ analytical decision support for improving efficiency,
+ reducing costs, and increasing profitability."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+# the Academic Initiative version (as used in this file) can be downloaded as described on
+# https://www.ibm.com/developerworks/community/blogs/jfp/entry/cplex_studio_in_ibm_academic_initiative?lang=en
+# a restricted "Community edition" version can be found on
+# https://www-01.ibm.com/software/websphere/products/optimization/cplex-studio-community-edition/
+sources = ['cplex_studio%s.linux_x86_64.bin' % ''.join(version.split('.'))]
+checksums = ['c8f8e6a2eb55433fddb5986c78ffe23cb4c6dfd09d6c24cebcf16d928a988a1d']
+
+builddependencies = [('binutils', '2.37')]
+
+dependencies = [
+    ('Java', '11.0.2', '', True),
```

we should depend on the `Java` wrapper, rather than a specific version directly
```suggestion
    ('Java', '11', '', SYSTEM),
```